### ISSUE TITLE
yolo_meta_arch

### ISSUE THREAD

**Issue by omg777** [rky0930/yolo_v2#2]
In model_builder, there is 
“from object_detection.meta_architectures import yolo_meta_arch”

but there is no yolo_meta_arch.
Can you help me ?

**Comment by rky0930** [comment]
Hi. @omg777 
There is yolo_meta_arch.py under research/object_detecion/meta_architectures
Clink this [link](https://github.com/rky0930/yolo_v2/blob/master/research/object_detection/meta_architectures/yolo_meta_arch.py)

**Comment by NaJaeMin92** [comment]
@rky0930 Thanks for your quick reply! I solved this problem but unfortunately, I got `AttributeError: 'SsdFeatureExtractor' object has no attribute 'batch_norm_trainable'` .
needs your help.

**Comment by omg777** [comment]
@rky0930 I solved this problem. thank you!
but unfortunately I got `AttributeError: 'SsdFeatureExtractor' object has no attribute 'batch_norm_trainable'
` error. Can help me?

**Comment by rky0930** [comment]
Becuase YOLOv2Darknet19FeatureExtractor inherit the ssd_meta_arch.SSDFeatureExtractor, they should have the same param set. 

Original repo's object detection api is under development. so, the source code is updated sometime. 
What Object detecion api source are you using now ?

**Comment by omg777** [comment]
@rky0930 I want to use YOLOv2, so I clone the original tensorflow repo and added your updated files to the original.
then I tried to train with my own dataset using YOLOv2.
 `python train.py --logtostderr --train_dir=training/ --pipeline_config_path=training/yolo_v2_darknet_19_voc.config --num_clones=2 --ps_tasks=1
`
but got a error `AttributeError: 'SsdFeatureExtractor' object has no attribute 'batch_norm_trainable'`

**Comment by rky0930** [comment]
@omg777 I'm not tried to use my code with newest Object Detection Model.
You can delete "batch_norm_trainable" from yolo_meta_arch.py,, but then maybe another error comes out. 
If you want to use Yolo_v2 with newest version of Object detection API, I will update my code. but it will take a time. 
If you ok with old version of object detection model, you can just clone this repo. 
If you want to do by yourself. Feel free to ask me :D

**Comment by omg777** [comment]
@rky0930 thanks a lot! 
I cloned your repo and tried to start training but still ‘batch_norm_trainable’ error exists.

**Comment by rky0930** [comment]
@omg777 Cloned source code still have the same error..That's weird.
Can i see your source code and config file ? If you can, send me via e-mail. I will check it by myself. 
E-mail: rky0930@gmail.com

**Comment by rky0930** [comment]
Hi @omg777 
Thank you for sending me your code.
I checked your source code. There was no 'batch_norm_trainable' error to me. 
I can't reproduce the error. But there was something i can change in your files to run with yolo_v2 correctly. 
I test train & eval with yolo_v2 & your code. **It runs well.** 

I replied your email and attached the files. 
If you have a question, Feel free to contact with.

**P.S. The changes are all about personal settings. There is no change in Yolo_v2 code base.**
